docs/dynamic_configuration.rst

```diff
@@ -1,46 +1,59 @@
+	Additionally, the node status also indicates this, by showing ``"restart_pending": true``.
 
-- startup configuration (also in patroni.yml).
-	They should be applied during the initialization time. Unlike other options, they are not written in etcd and
-	any attempts to change them dynamically are blocked.
+- Local `configuration <https://github.com/zalando/patroni/blob/master/docs/SETTINGS.rst>`__ (patroni.yml).
+	Those options are defined in the configuration file and take precedence over dynamic configuration.
+	patroni.yml could be changed and reload in runtime (without restart of Patroni) by sending SIGHUP to the Patroni process or by performing ``POST /reload`` REST-API request.
 
-- dynamic configuration.
-	Those options can be set in etcd at any time. If the options changed are not part of the startup configuration,
-	they are applied asynchronously (upon the next wake up cycle) to every node, which gets subsequently reloaded.
-	If the node requires a restart to apply the configuration (for options with context postmaster or internal, if
-	their values have changed), a special flag indicating this should be set in the members.data JSON. A new API
-	endpoint should return whether the given node requires a restart. Additionally, the node status should also
-	indicate this.
+- Environment `configuration <https://github.com/zalando/patroni/blob/master/docs/ENVIRONMENT.rst>`
+	It is possible to set/override some of the "Local" configuration parameters with environment variables.
+	Environment configuration is very useful when you are running in dynamic environment and you don't know some of the parameters in advance (for example it's not possible to know you external IP address when you are running inside ``docker``).
+
+Some of PostgreSQL parameters must be set to the same value on master and replicas and therefore controlled only via "Dynamic configuration". Any attemt to change them via "Local configuration" are blocked:
 
-Some options may not be increased on the master independently of the replicas (master-dependent options):
+- max_connections: 100
+- max_locks_per_transaction: 64
+- max_worker_processes: 8
+- max_prepared_transactions: 0
+- wal_level: hot_standby
+- wal_log_hints: on
+- track_commit_timestamp: off
 
-- max_connections
-- max_locks_per_transactions
-- max_worker_processes
-- max_prepared_transactions
+Although following parameters are not necessarily must be the same on master and replicas, but it is better to have them eqals, therefore they are also controlled by Patroni and can be set only via Dynamic configuration.
+- max_wal_senders: 5
+- max_replication_slots: 5
+- wal_keep_segments: 8
 
-Regarding the options that can be set, the following restrictions apply:
+Important! Patroni does some simple checks of new values obtained from DCS before applying them. Above lists containing some sane default values and it is not allowed to set new value smaller.
```

Comment:
I'd change this to:

"NB: You won't be able to decrease the values for the numerical parameters described in this section below the default ones or disable the boolean flags set by default to on. The default values are set permit PostgreSQL to run the replication with several replicas.'